admin/react/assets/src/dashboard-app/pages/fonts/edit/preview/EditLocalPreviewItem.js

```diff
@@ -0,0 +1,61 @@
+import React from "react";
+import { useSelector } from 'react-redux';
+import { __ } from "@wordpress/i18n";
+import LFontVariation from '../../preview/LFontVariation';
+
+const EditLocalPreviewItem = (fontId) => {
+	const editFont = useSelector( ( state ) => state.editFont );
+
+	if ( undefined === editFont || null === editFont ) {
+		return;
+	}
+
+	if ( ! editFont.variations.length || '' === editFont.font_name ) {
+		return;
+	}
+
+	const variations = editFont.variations;
+	const fontName = editFont.font_name;
+
+	const getLocalFontStyle = () => {
+		let defaultFont = `@font-face {\r\n\tfont-family: '${fontName}';\r\n\tfont-style: normal;`;
+		let srcFont = '';
+
+		variations.map((variation) => {
+			let fontUrl = variation.font_url,
```

Comment:
I have noticed multiple places this kind of a function is used. So rather then repeating it we can just make a common function and reuse this function @imnavanath , Please check other places too

Reply:
Added on board - https://brainstormforce.atlassian.net/browse/CFP-40 assigned to Babu